src/lib/const.ts

```diff
@@ -337,29 +337,29 @@ export const SEPARATOR = `\n\n${color.yellow(
   `^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^`
 )}\n\n`;
 
-const MAIN_AQUA_FILE_STATUS_TEXT = `export status
-
--- example of running a service deployed using 'fluence deal deploy'
+const MAIN_AQUA_FILE_STATUS_TEXT = `
+-- example of running services deployed using 'fluence deal deploy'
 -- with worker '${DEFAULT_WORKER_NAME}' which has service 'MyService' with method 'greeting'
 
-func status():
-    workersInfo <- getWorkersInfo()
-    dealId = workersInfo.deals.${DEFAULT_WORKER_NAME}.dealId
-    print = (answer: string, peer: string):
-      Console.print([answer, peer])
+export runDeployedServices
 
-    answers: *string
+data Answer:
+    answer: string
+    peer: string
+
+func runDeployedServices() -> *Answer:
```

Comment:
You have the same code in `test/aqua/runDeployedWorkers.aqua`.

Is it possible to DRY that and keep only separate `.aqua` file?